lib/monitor/monitor.go

```diff
@@ -18,31 +18,51 @@ import (
 var Sink = "0.0.0.0"
```

Comment:
This is the state of the nation:

- Measure - is what is used in the clients to start a measure
- Value - one measurement
- Measurement - Wall/User/System measurement tuple
- Stats - slice of Measurements

But it's confusing, to say the least...